server/src/test/java/org/elasticsearch/search/aggregations/metrics/ScriptedMetricIT.java

```diff
@@ -365,52 +358,12 @@ public void testMap() {
         assertThat(numShardsRun, greaterThan(0));
     }
 
-    public void testExplicitAggParam() {
```

Comment:
This case is not meaningful anymore since `state` is always implicit on the context.